src/metabase/notification/payload/core.clj

```diff
@@ -36,18 +37,8 @@
         [:event_info  [:maybe :map]]]]]]
     [:notification/card
      [:map
-      ;; replacement of pulse
-      [:alert      [:map
-                    [:card_id                           ms/PositiveInt]
-                    [:schedule                          :map]
-                    [:alert_condition  {:optional true} [:enum "rows" "goal"]]
-                    [:alert_above_goal {:optional true} [:maybe ms/BooleanValue]]
-                    [:alert_first_only {:optional true} [:maybe ms/BooleanValue]]
-                    [:include_csv      {:optional true} [:maybe ms/BooleanValue]]
-                    [:include_xls      {:optional true} [:maybe ms/BooleanValue]]
-                    [:format_rows      {:optional true} [:maybe ms/BooleanValue]]
-                    [:pivot_results    {:optional true} [:maybe ms/BooleanValue]]]]
-      [:creator_id ms/PositiveInt]]]
+      [:payload    {:optional true} models.notification/NotificationCard]
```

Comment:
changed from :alert to :payload

Reply:
this is the reason for most changes in the PR, everything else has to follow this:
- all channel rendering code has to replace alert with this
- send pulse code has to transform alert to payload